tracer/src/Datadog.Trace/SpanContextPropagator.cs

```diff
@@ -159,12 +159,22 @@ public SpanContext Extract<T>(T carrier, Func<T, string, IEnumerable<string>> ge
             return new SpanContext(traceId, parentId, samplingPriority, null, origin);
         }
 
-        public IEnumerable<KeyValuePair<string, string>> ExtractHeaderTags<T>(T headers, IEnumerable<KeyValuePair<string, string>> headerToTagMap, string defaultTagPrefix)
+        public IEnumerable<KeyValuePair<string, string>> ExtractHeaderTags<T>(T headers, IEnumerable<KeyValuePair<string, string>> headerToTagMap, string defaultTagPrefix, string useragent = null)
```

Comment:
Personally I'd rather create this as an overload rather than use default args, but there's definitely a lot of precedent for your approach 😉

Reply:
Yes I have no big preference. I'll change it